support-modules/rest/src/main/scala/com/gu/rest/WebServiceHelper.scala

```diff
@@ -97,7 +97,7 @@ trait WebServiceHelper[Error <: Throwable] {
         .toTry
       responseBody <- Try(body.string())
       codeBody = CodeBody(code, responseBody)
-      _ = SafeLogger.info(s"response $code body: ${responseBody}")
+      _ = SafeLogger.info(s"response $code body: ${responseBody.replace("\n", "")}")
```

Comment:
another option which works for EC2 instance logging to cloudwatch (but not laambdas) is to indent the continuation lines with whitespace
```suggestion
      _ = SafeLogger.info(s"response $code body: ${responseBody.replace("\n", "\n ")}")
```

Reply:
I suspected so. I haven't found a way to make it work with lambdas.